CHANGES

```diff
@@ -2,10 +2,51 @@ SUMMARY OF RELEASE CHANGES FOR GREENBONE VULNERABILITY MANAGER
 ==============================================================
 
 For detailed code changes, please visit
- https://github.com/greenbone/gvm/commits/master
+ https://github.com/greenbone/gvmd/commits/master
 or get the entire source code repository and view log history:
-$ git clone https://github.com/greenbone/gvm.git
-$ cd gvm && git log
+$ git clone https://github.com/greenbone/gvmd.git
+$ cd gvmd && git log
+
+gvmd 8.0+beta2 (2018-12-04)
+
+This is the second beta release of the gvmd module 8.0 for the Greenbone
+Vulnerability Management (GVM) framework.
+
+It was renamed from gvm to gvmd (greenbone vulnerability manager daemon).
+
+Apart from the new name, the module covers a number of significant advances
+and clean-ups compared to the previous version.
+
+Many thanks to everyone who has contributed to this release:
+Hani Benhabiles, Christian Fischer, Matthew Mundell, Timo Pollmeier,
+Bjoern Ricks, Jan-Oliver Wagner and Michael Wiegand.
+
+Main changes compared to gvm 8.0+beta1:
+* Classic report format HTML has been removed from predefined report formats.
+* External tool openvasmr has been renamed to gvmcg.
+* Size of result description and diff text in GMP has been limited.
+* Support for creating filters for vulnerabilities has been added.
+* .deb and .rpm LSC package generation has been improved.
+* SNMP and ESXi Authorization in the scan config have been hidden.
+* Schedules have been improved.
```

Comment:
This sounds a bit vague. What is the improvement about? If you're not sure, please give me the PR you're referring to here.